docs/user/managing-devices.md

```diff
@@ -417,6 +453,53 @@ spec:
 [...]
 ```
 
+## Scheduling Updates and Downloads
```

Comment:
Would you mind adding this new section to the TOC in the README, please?
Should we move this section down in the TOC so it's not read as being limited to OS only?